cvat-ui/src/components/annotation-page/canvas/views/canvas2d/canvas-wrapper.tsx

```diff
@@ -483,13 +483,27 @@ class CanvasWrapperComponent extends React.PureComponent<Props> {
         if (prevProps.highlightedConflict !== highlightedConflict) {
             const severity: HighlightSeverity | undefined = highlightedConflict
                 ?.severity as unknown as HighlightSeverity;
-            const highlightedClientIDs = (highlightedConflict?.annotationConflicts || [])
+            const highlightedShapeClientIDs = (highlightedConflict?.annotationConflicts || [])
                 .map((conflict: AnnotationConflict) => annotations
-                    .find((state) => state.serverID === conflict.serverID && state.objectType === conflict.type),
+                    .find((state) => conflict.type !== 'tag' && state.serverID === conflict.serverID && state.objectType === conflict.type),
                 ).filter((state: ObjectState | undefined) => !!state)
                 .map((state) => state?.clientID) as number[];
 
-            canvasInstance.highlight(highlightedClientIDs, severity || null);
+            // if current shape conflict ids is empty then there might be a conflict in tag annotations
+            if (highlightedShapeClientIDs.length === 0) {
```

Comment:
It seems this logic breaks removing highlighting from the objects. 
I suppose this code is better to be written this way:
1. Collect clientIDs for all shape types, even for tags. But create a bool variable that will be true if the conflict is tag.
2. Then if this variable is true, dont do anything. If false, pass the  highlightedShapeClientIDs to the canvasInstance (even empty array for correct de-highlighting)